src/engine/source/builder/environment.cpp

```diff
@@ -125,18 +125,18 @@ std::string Environment::getGraphivzStr()
     auto removeHyphen = [](const std::string& text)
     {
         auto ret = text;
-        auto pos = ret.find("-");
+        auto pos = ret.find('-');
         while (pos != std::string::npos)
         {
             ret.erase(pos, 1);
-            pos = ret.find("-");
+            pos = ret.find('-');
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

'find' called with a string literal consisting of a single character; use the more effective overload accepting a character